index.html

```diff
@@ -167,7 +167,39 @@ <h3 class="title">Skills</h3>
         </ul>
       </div>
     </section>
+
   </main>
+  <footer class="footer">
+    <div class="footer-shapes"></div>
+    <div class="footer-content">
+      <div class="contact-intro">
+        <div class="title">
+          Contact me
+        </div>
+        <div class="description">
+          If you have an application you are interested in developing, a feature that you need built or a project that
+          needs coding. I'd love to help with it
+        </div>
+      </div>
+      <form action="https://formspree.io/f/xbjwdrql" method="post" class="contact-form">

```

Comment:
Nice work applying the formspree functionality 😎

The form has been sent successfully ✔
![Captura](https://user-images.githubusercontent.com/15807343/149219966-e83dd6d6-2f4a-42df-9577-1daaa18936fa.PNG)